servers/Azure.Mcp.Server/README.md

```diff
@@ -1,150 +1,298 @@
-    - [Complete List of Supported Azure Services](#complete-list-of-supported-azure-services)
-- [Support & Reference](#support-and-reference)
-    - [Documentation](#documentation)
-    - [Feedback & Support](#feedback-and-support)
-    - [Security](#security)
-    - [Data Collection](#data-collection)
-    - [Contributing & Code of Conduct](#contributing)
+To generate a README tailored for a specific package type, run:
 
-# <a id="overview"></a> Overview
+eng\scripts\Process-PackageReadMe.ps1 -InputReadMePath "<path to this readme>" -OutputDirectory "<output directory>" -PackageType "<nuget|npm|vsix>"
 
-**Azure MCP Server** supercharges your agents with Azure context across **30+ different Azure services**.
+## Supported Annotations
 
-# <a id="installation"></a> Installation
+All annotations must be wrapped in markdown comments . The following annotations are supported:
 
-## <a id="ide-extensions"></a> 🧩 IDE Extensions
+### 1. Section Removal
 
-Follow these simple steps to start using Azure MCP with your favorite IDE.  We recommend VS Code:
+**Purpose:** Remove one or more lines, or parts of a line of markdown for specified package types.
 
-### <a id="vs-code-recommended"></a> 🔷 VS Code (Recommended)
+**Syntax:**
+remove-section: start packageType1;packageType2
 
-1. Install either the stable or Insiders release of VS Code:
-   * [💫 Stable release](https://code.visualstudio.com/download)
-   * [🔮 Insiders release](https://code.visualstudio.com/insiders)
-1. Install the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions
-1. Install the [Azure MCP Server](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-mcp-server) extension
+e.g.
+remove-section: start nuget;vsix;npm
+......
+various markdown lines
+.....
+remove-section: end
 
-### <a id="visual-studio-2022"></a> 💜 Visual Studio 2022
+### 1. Insert Chunk
 
-From within Visual Studio 2022 install [GitHub Copilot for Azure (VS 2022)](https://marketplace.visualstudio.com/items?itemName=github-copilot-azure.GitHubCopilotForAzure2022):
-1. Go to `Extensions | Manage Extensions...`
-2. Switch to the `Browse` tab in `Extension Manager`
-3. Search for `Github Copilot for Azure`
-4. Click `Install`
+insert-chunk is used to insert a chunk of text into a line for a specified package type. 
+e.g.
+insert-section: nuget;vsix;npm {{Text to be inserted}}
 
-### <a id="intellij-idea"></a> ☕ IntelliJ IDEA
+Remember to wrap each annotation in markdown comments
+-->
+# <!-- remove-section: start nuget;vsix --><img height="36" width="36" src="https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/acom_social_icon_azure" alt="Microsoft Azure Logo" /> <!-- remove-section: end -->Azure MCP Server <!-- insert-section: nuget;vsix;npm {{ToolTitle}} -->
```

Comment:
I'm still not a fan that insert uses different syntax than remove. I get it might be a little less verbose but I think it will cause more confusion longer term and make inserting larger sections much harder one line at a time.

Reply:
Isn't that also true for the remove section?